.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.py]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
You should add the `insert_final_newline` for other files (like .json) too. Not using a final new line is annoying with git diffs.